app/src/main/java/com/github/se/polyfit/ui/components/selector/MealSelector.kt

```diff
@@ -82,7 +82,7 @@ fun MealSelector(
                   modifier = Modifier.weight(1f),
               )
               Text(
-                  text = "${selectedMeal.calculateTotalCalories()} kcal",
+                  text = "${selectedMeal.calculateTotalCalories().roundTwoDecimal()} kcal",
```

Comment:
it could be nice to use the String ressource file to pass your strings so something like this

```
context.getString(R.string.calories, selectedMeal.calculateTotalCalories().roundTwoDecimal())
```
and then in the string you could put something like 
```
    <string name="calories">%1$.2f kCal</string>
```